staging/src/kubevirt.io/api/core/v1/types.go

```diff
@@ -2191,17 +2191,41 @@ type SMBiosConfiguration struct {
 
 // MigrationConfiguration holds migration options
 type MigrationConfiguration struct {
-	NodeDrainTaintKey                 *string            `json:"nodeDrainTaintKey,omitempty"`
-	ParallelOutboundMigrationsPerNode *uint32            `json:"parallelOutboundMigrationsPerNode,omitempty"`
-	ParallelMigrationsPerCluster      *uint32            `json:"parallelMigrationsPerCluster,omitempty"`
-	AllowAutoConverge                 *bool              `json:"allowAutoConverge,omitempty"`
-	BandwidthPerMigration             *resource.Quantity `json:"bandwidthPerMigration,omitempty"`
-	CompletionTimeoutPerGiB           *int64             `json:"completionTimeoutPerGiB,omitempty"`
-	ProgressTimeout                   *int64             `json:"progressTimeout,omitempty"`
-	UnsafeMigrationOverride           *bool              `json:"unsafeMigrationOverride,omitempty"`
-	AllowPostCopy                     *bool              `json:"allowPostCopy,omitempty"`
-	DisableTLS                        *bool              `json:"disableTLS,omitempty"`
-	Network                           *string            `json:"network,omitempty"`
+	// NodeDrainTaintKey defines the taint key that indicates a node should be drained.
+	// Note: this option relies deprecated node taint feature. Default: kubevirt.io/drain
+	NodeDrainTaintKey *string `json:"nodeDrainTaintKey,omitempty"`
+	// ParallelOutboundMigrationsPerNode is the maximum number of concurrent outgoing live migrations
+	// allowed per node. Defaults to 2
+	ParallelOutboundMigrationsPerNode *uint32 `json:"parallelOutboundMigrationsPerNode,omitempty"`
+	// ParallelMigrationsPerCluster it the total number of concurrent live migrations
+	// allowed cluster-wide. Defaults to 5
+	ParallelMigrationsPerCluster *uint32 `json:"parallelMigrationsPerCluster,omitempty"`
+	// AllowAutoConverge allows the platform to compromise performance/availability of VMIs to
+	// guarantee successful VMI live migrations. Defaults to false
+	AllowAutoConverge *bool `json:"allowAutoConverge,omitempty"`
+	// BandwidthPerMigration limits the amount of network bandwith live migrations are allowed to use.
+	// The value is in MiB per second. Defaults to 0 (no limit)
```

Comment:
> The value is in MiB per second

I don't think that's true :)

This is actually of type `resource.Quantity`, so for 1MiB the desired value is `"1Mib"`, but the value can also be in KiB, etc.

Reply:
Thanks, fixed